src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1237,6 +1237,8 @@ public virtual MethodSymbol TupleUnderlyingMethod
             }
         }
 
+        public virtual Location ErrorLocation => Locations[0];
```

Comment:
>public [](start = 8, length = 6)

We usually mark internal symbol APIs as internal even though the type is not public. This explicitly states the intent and prevents accidentally implementing some interface through implicit implementation.